Grand.Web/Areas/Admin/Views/Shared/Menu.cshtml

```diff
@@ -18,6 +18,29 @@
         <div class="main_menu_side hidden-print main_menu" id="sidebar-menu">
             <div class="menu_section">
                 <ul class="nav side-menu page-sidebar-menu">
+                    <li class="sidebar-search-wrapper" id="SearchLi">
```

Comment:
May I ask you what's the purpose of adding it to the GrandNode?

Reply:
to add search textbox to admin panel menu allowing the user to fast filter menu to search what ever he want

Reply:
But it's possible with the admin search on top.

Reply:
you are right, i didn't notice this search . i think i had the idea from previous version but added it now so i couldn't see that you have implemented a better solution. you can rollback this change

Reply:
Please remove that changes from pull request and send it again.